opennms-model/src/main/java/org/opennms/netmgt/model/ResourcePath.java

```diff
@@ -54,27 +57,28 @@
     private final List<String> elements = new ArrayList<>();
 
     public ResourcePath(String... path) {
-        for (String el : path) {
-            elements.add(el);
-        }
+        this(Arrays.asList(path));
     }
 
     public ResourcePath(Iterable<String> pathElements) {
         for (String el : pathElements) {
+            if(el != null && el.contains("/")) {
+                throw new IllegalArgumentException("path elements must not contain a forward slash");
```

Comment:
@j-white I try to make here explicit that we can't have a forward slash in a path. I hope this won't have side effects?

Reply:
Should be OK.

Let's include the path elements in the exception message if thrown - will allow us to trace down the culprit.

Reply:
fixed